examples/streaming_transcribe_from_local_file.js

```diff
@@ -27,8 +27,15 @@ client.on('connect', connectionMessage => {
     console.log(`Connected with job id: ${connectionMessage.id}`);
 })
 
+// Optional config to be provided.
+const sessionConfig = new revai.SessionConfig(
+    /* (optional) metadata */ "this is an example", 
+    /* (optional) custom_vocabulary_id */ null, 
+    /* (optional) filter_profanity */ true
+);
```

Comment:
Very minor - would it better to switch the places of the argument and the comment? Like
```suggestion
// Optional config to be provided.
const sessionConfig = new revai.SessionConfig(
    "this is an example",  /* (optional) metadata */ 
    null,                  /* (optional) custom_vocabulary_id */ 
    true                   /* (optional) filter_profanity */
);
```

Reply:
I'll check to see if it looks better in the IDEs.